lib/ui/platform_dispatcher.dart

```diff
@@ -73,6 +73,23 @@ const String _kFlutterKeyDataChannel = 'flutter/keydata';
 ByteData? _wrapUnmodifiableByteData(ByteData? byteData) =>
     byteData == null ? null : UnmodifiableByteDataView(byteData);
 
+/// A token that represents a root isolate.
+class RootIsolateToken {
+  RootIsolateToken._(this._token);
+
+  final int _token;
```

Comment:
just a nit, consider adding a doc comment here explaining what this is, even though it's not public.

Reply:
done